test/net/sourceforge/kolmafia/request/UseSkillRequestTest.java

```diff
@@ -121,5 +124,29 @@ void doNotEquipDesignerSweatpantsForSkillIfAlreadyWearing() {
             requests.get(0), "/runskillz.php", "action=Skillz&whichskill=7419&ajax=1&quantity=1");
       }
     }
+
+    @Test
+    void decreaseSweatWhenCastingSweatBooze() {
+      Preferences.setInteger("sweat", 31);
+      UseSkillRequest.lastSkillUsed = SkillPool.SWEAT_OUT_BOOZE;
+      UseSkillRequest.lastSkillCount = 1;
+      UseSkillRequest.parseResponse(
+          "runskillz.php?action=Skillz&whichskill=7414&ajax=1&quantity=1",
+          html("request/test_cast_sweat_booze.html"));
+      // 31 - 25 = 6
+      assertEquals(Preferences.getInteger("sweat"), 6);
+    }
+
+    @Test
+    void decreaseSweatWhenCastingOtherSweatSkills() {
+      Preferences.setInteger("sweat", 69);
+      UseSkillRequest.lastSkillUsed = SkillPool.DRENCH_YOURSELF_IN_SWEAT;
```

Comment:
One last thing, could you reset these in an AfterAll or something? We should probably have a Cleanup for them of course...

Reply:
A cleanup for sweat. Like a towel? :)